cpp/velox/memory/VeloxMemoryManager.cc

```diff
@@ -316,22 +316,27 @@ bool VeloxMemoryManager::tryDestructSafe() {
 
   // Velox memory manager considered safe to destruct when no alive pools.
   if (veloxMemoryManager_) {
-    if (veloxMemoryManager_->numPools() > 1) {
+    if (veloxMemoryManager_->numPools() > 2) {
```

Comment:
note this is due to 
https://github.com/facebookincubator/velox/commit/763c19c0c767035ce03573ca2dc852b1303eeb6f